### DISCUSSION TITLE
duda sobre los videos de contenido extras en teams

### DISCUSSION THREAD

**Discussion by FranciscoSantos312** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#166] [Q&A]
hola donde puedo encontrar en teams los videos del contenido extra del MC  donde explica un poco mas sobre los temas vistos. solo me aparecen los lives pero me confunde teams y no los encuentro :(

**Comment by OmarVenturaP** [comment]
Que tal compañero, puedes revisar en el repositorio de Frontend, allí donde están los temarios se habilito una carpeta con el temario, allí dentro encontraras los links a los diferentes videos explicativos.

**Reply by AlexisRamirezHernandez** [reply]
Hola amigo aqui encontraras todo lo que corresponde a la mision, solo identifica el tema que desees, haces clic y ahi viene.  [Soy el enlace a el repositorio principal](https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision)
Para los temás que comentas puedes hacer clic aqui: 
[Link a los contenidos](https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/02%20-%20HTML/temario)
Todo viene en el repositorio de la mision

**Comment by doguedogue** [comment]
Están aquí mira, se pueden filtrar los creados por el MC
**NOTA**: Recuerda estar loggeado a tu cuenta @innovaccion en office/teams

https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a

![image](https://user-images.githubusercontent.com/23409026/156209663-2e0ac53b-9a23-47d7-b728-d5abe0230885.png)
